OOP/Assignment 1/OOPQuestion1/Day5TestJUnit.java

```diff
@@ -0,0 +1,40 @@
+import java.util.HashMap;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+
+public class Day5TestJUnit
+{
+
+	@Test
+	public void test()
+	{
+		ServiceStation serviceStation = new ServiceStation();
+		serviceStation.readMechanics();
+		serviceStation.readCars();
+		serviceStation.doAllotment();
+		HashMap<String, Integer> carProcessingList = serviceStation.getCarProcessingStatus();
```

Comment:
we would be needing to convert object into specific class type we use just MAP instead of generic one